app/controllers/api/v1/trucking/driver_commissions_history_controller.rb

```diff
@@ -0,0 +1,25 @@
+module Api
+  module V1
+    module Trucking
+      # Driver Commission History Api endpoint
+      class DriverCommissionsHistoryController < ApplicationController
+        # GET /driver_commission_history.json
+        def index
+          respond_to do |format|
+            format.json { render :json => DriverCommissionHistoryAdapter.all }
+          end
+        end
+
+        # GET /driver_commission_history#by_paid_date
+        def by_paid_date
+          respond_to do |format|
+            format.json do
+              render :json => DriverCommissionHistoryAdapter
+                .by_paid_date(params['paid_date'])
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.